packages/webapp/src/hooks/state/authentication.tsx

```diff
@@ -32,7 +32,11 @@ export const useAuthActions = () => {
 
       removeAuthenticationCookies();
 
-      window.location.reload();
+      if (href) {
+        window.location.href = href;
+      } else {
+        window.location.reload();
+      }
```

Comment:
@abouolia Here the only change is, if the session_end url is passed, it will be redirected to that url and gets callback from OIDC provider to post logout uri else just reload the page.